cli/js/umask.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+import { sendSync } from "./dispatch_json.ts";
+
+/**
+ * **UNSTABLE**: maybe needs `allow-env` permissions.
+ *
+ * If `mask` is provided, sets the process umask. Always returns what the umask
+ * was before the call.
+ */
+export function umask(mask?: number): number {
+  return sendSync("op_umask", { mask });
+}
```

Comment:
This belongs in `cli/js/ops/fs.ts` (which doesn't exist at time of writing; we're in the middle of a reorg). @bartlomieju where do you suggest he place this right now?

Also we need a test of this op...

Reply:
test is coming...

Reply:
> This belongs in `cli/js/ops/fs.ts` (which doesn't exist at time of writing; we're in the middle of a reorg). @bartlomieju where do you suggest he place this right now?
> 
> Also we need a test of this op...

`cli/js/ops/fs.rs` would be best, but I'm fine with landing as is - I'd rather reorg all `fs` ops in one go